### DISCUSSION TITLE
Hpw do I track down this enter tree error?

### DISCUSSION THREAD

**Discussion by blackears** [godotengine/godot-proposals#8822] [Scripting]
I'm trying to track down an error that is printing this to the console:

```
core/math/basis.cpp:47 - Condition "det == 0" is true.
core/math/transform_2d.cpp:51 - Condition "det == 0" is true.
```

After running godot from source in Debug mode, it seems to be happening when a mouse hover operation tries to run just after a TabBar is added to the scene.  I'm guessing the CanvasItem has zero area at this point, which is causing an affine_inverse operation to fail.  I have no idea which component is being added to my tree, though, which is what's causing this.  Is there any way to get the path of the node or maybe the path of the node being added to?

![image](https://github.com/godotengine/godot-proposals/assets/6724608/12f7037c-da42-4cff-87f2-19fb55da7560)

**Comment by AThousandShips** [comment]
I'd suggest asking in the other community channels, this is for feature discussions, not support questions

**Comment by AThousandShips** [comment]
Closing as per the above, please use the appropriate channels to ask support questions 🙂

**Comment by Calinou** [comment]
- Superseded by https://github.com/godotengine/godot/issues/86915.
